deploy/aws/variables.tf

```diff
@@ -83,3 +83,24 @@ variable "tikv_count" {
 variable "tidb_count" {
   default = 2
 }
+
+variable "pd_instance_type" {
```

Comment:
Maybe we should add a warning here, since it's fragile to change instance type which may break the user-data and local volume setup.

Reply:
Added a comment in `variables.tf` and also in tutorial doc.